pkg/engine/jsonContext.go

```diff
@@ -161,6 +162,18 @@ func fetchImageDataMap(ref string) (interface{}, error) {
 	return untyped, nil
 }
 
+func fetchImageDigest(ref string) (string, error) {
```

Comment:
Why is this in `jsonContext`? Would it be best to declare in imageVerify where its used?